ipn/message.go

```diff
@@ -56,8 +56,8 @@ type FakeExpireAfterArgs struct {
 }
 
 type PingArgs struct {
-	IP      string
-	UseTSMP bool
+	IP   string
+	Type string
```

Comment:
I think this isn't used by other ipn API clients, so the change is ok, but do we have any additional process or considerations for such changes? Are there things I should do to help me check I didn't break anything?

Reply:
we don't generally try too hard to be compatible at this layer. (we generally don't support mix & matching versions of `cmd/tailscale` and `tailscaled`)